Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/refine_mesh_at_isolevel.h

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2021 GeometryFactory (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+//
+// Author(s)     : Sebastien Loriot
+
+#ifndef CGAL_POLYGON_MESH_PROCESSING_REFINE_MESH_AT_ISOLEVEL_H
+#define CGAL_POLYGON_MESH_PROCESSING_REFINE_MESH_AT_ISOLEVEL_H
+
+#include <CGAL/license/Polygon_mesh_processing/miscellaneous.h>
+
+#include <CGAL/boost/graph/named_params_helper.h>
+
+namespace CGAL {
+namespace Polygon_mesh_processing {
+namespace experimental {
+
+/*! \ingroup PkgPolygonMeshProcessingRef
+ * Function object that computes the intersection of a plane with
```

Comment:
Completely wrong paragraph.  Not a function object and not intersecting with a plane.
Also I am wondering why I do not see the function in the doc built in the PR.